src/components/menu/SubmenuItem.tsx

```diff
@@ -3,6 +3,7 @@ import classNames from 'classnames';
 import omit from 'lodash/omit';
 
 import './SubmenuItem.scss';
+import Arrow16 from 'box-ui-elements/es/icon/fill/Arrow16';
```

Comment:
I don't think this import works. Since we're inside the box-ui-elements project, we should reference the icon like this: `../../icon/fill/Arrow16`